app/pages/delete-order/template.test.js

```diff
@@ -0,0 +1,68 @@
+import { componentTester } from '../../test-utils/componentTester';
+import manifest from './manifest.json';
+
+const setup = {
+  template: {
+    path: 'pages/delete-order/template.njk',
+  },
+};
+
+const context = {
+  ...manifest,
+  backLinkHref: '/organisation/order-1',
+  orderDescription: 'some description',
+};
+
+describe('delete order page', () => {
+  it('should render a backLink', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const backLink = $('[data-test-id="go-back-link"]');
+      expect(backLink.length).toEqual(1);
+      expect(backLink.text().trim()).toEqual('Go back');
+      expect($(backLink).find('a').attr('href')).toEqual('/organisation/order-1');
+    });
+  }));
+
+  it('should render the delete order page title', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const title = $('h1[data-test-id="delete-order-page-title"]');
+      expect(title.length).toEqual(1);
+      expect(title.text().trim()).toEqual(context.title);
+    });
+  }));
+
+  it('should render the delete order page description', componentTester(setup, (harness) => {
+    harness.request(context, ($) => {
+      const description = $('h2[data-test-id="delete-order-page-description"]');
+      expect(description.length).toEqual(1);
+      expect(description.text().trim()).toEqual(context.description);
+    });
+  }));
+
+  it('should render the delete order order details', componentTester(setup, (harness) => {
```

Comment:
Sam & I decided to split this test out. Apologies we should have probably told you or updated the other examples. But we agreed & would be good to get your opinion that is it better to split the assertions out into two separate tests, hopefully making them easier to read & quickly understand a failure. One for the order description title & one for the order description.

Reply:
I agree!